### ISSUE TITLE
Extending Favorites functionality to more Things.

### ISSUE THREAD

**Issue by alexatrhea** [RHEAGROUP/CDP4-IME-Community-Edition#249]
Favoriting and filtering has been added to ParameterType Browser and can be potentially expanded to other concepts like `Category` for instance. Investigate which views can benefit from this and implement.

**Comment by samatrhea** [comment]
I agree with the approach. I do propose we limit this to Reference Data and not get bogged down with model data. I see value in the following types:
  - categories
  - constants
  - rules
